### ISSUE TITLE
An admin should be able to archive an output / output indicator

### ISSUE THREAD

**Issue by samjamead** [Blue-Marine-Foundation/Maerl#61]
Archiving should remove the output / output indicator from the logframe but all impact data should still count towards the project's lifetime. 

Success criteria: 
- An archived output / output indicator should be hidden from the main TOC and logframe views 
- A list of archived output / output indicators should be available on each project 
- Any impact data relevant to the project should be retained 
- Any impact data relevant should be visible on any project summaries, i.e. when #56 goes live

**Comment by samjamead** [comment]
Hi @sunehajagan after our conversation earlier I am confident that this is the correct solution to the problem – would you like me to prioritise working on this in April?

**Comment by sunehajagan** [comment]
Yes! Shall we catch up this Friday to go over the Maerl prioritization list? Let me know what times work for you.

Best,
Suneha
________________________________
